grafana/map.jinja

```diff
@@ -4,6 +4,9 @@ Debian:
   pkgs:
   - grafana
   service: grafana-server
+  paths:
```

Comment:
Should be singular to match conventions (dict = singular, list = plural).
Also it's usually used `dir` as a key instead of `path`